tests/Browser/Pages/Backend/Product/CreateProductTest.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+namespace Tests\Browser\Pages\Backend\Product;
+
+use Tests\DuskTestCase;
+use Laravel\Dusk\Browser;
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+use App\Category;
+
+class CreateProductTest extends DuskTestCase
+{
+
+    use DatabaseMigrations;
+    const NUMBER_RECORD_CREATE = 5;
+    /**
+    * Override function setUp() for make product login
+    *
+    * @return void
+    */
+    public function setUp()
+    {
+        parent::setUp();
+        factory(Category::class, 'parent', self::NUMBER_RECORD_CREATE)->create();
+        factory(Category::class, self::NUMBER_RECORD_CREATE)->create();
+    }
+
+    /**
+     * A Dusk test route.
+     *
+     * @return void
+     */
+    public function testCreateRoute()
+    {
+        $this->browse(function (Browser $browser) {
+            $browser->visit('/admin/product/create')
+                    ->assertSee('Create Form', 'Quantity');
+        });
+    }
+
+    /**
+     * List case for Test validate for input Create Product
+     *
+     * @return array
+     */
+    public function listCaseValidation()
+    {
+        return [
+            ['name', '', 'The name field is required.'],
+            ['price', '', 'The price field is required.'],
+            ['quantity', '', 'The quantity field is required.'],
+            ['preview', '', 'The preview field is required.'],
+            ['description', '', 'The description must be a string.'],
+            ['images', '', 'The images field is required.'],
+        ];
+    }
+
+    /**
+     * Dusk test validate for input
+     *
+     * @param string $name name of field
+     * @param string $content content
+     * @param string $message message show when validate
+     *
+     * @dataProvider listCaseValidation
+     *
+     * @return void
+     */
+    public function testValidateForInput($name, $content, $message)
+    {
+        $this->browse(function (Browser $browser) use ($name, $content, $message) {
+            $browser->visit('admin/product/create')
+                    ->press('submit')
+                    ->assertSee($message);
+        });
+    }
+
+    /**
+     * Test Validation Create Product success.
+     *
+     * @return void
+     */
+    public function testCreateProductSuccess()
+    {
+        $this->browse(function (Browser $browser) {
+            $browser->visit('/admin/product/create')
+                    ->type('name', 'test name')
+                    ->type('price', '2.05')
+                    ->type('quantity', '20')
+                    ->select('category_id', '2')
+                    ->type('preview', 'aaa')
+                    ->type('description', 'aaa')
+                    ->attach('images[]', 'public/images/products/default-product.jpg')
+                    ->press('submit')
+                    ->pause(1000)
```

Comment:
Trường hợp sau 1 giây mà page vẫn chưa load xong thì sao?
Tham khảo: https://laravel.com/docs/5.6/dusk#waiting-for-elements